### PR TITLE
add details on googleComputeEngineR features and Cloud Run

### PR DESCRIPTION
**Pull request by MarkEdmondson1234**
#39 #37

### CODE REVIEW

File: manuscript.Rmd [hunk 1]
```diff
@@ -299,8 +299,8 @@ The cloud is the natural environment of containers, and becomes the go-to mechan
...
 - `RSelenium`
-- [`googleComputeEngineR`](https://cloudyr.github.io/googleComputeEngineR/) (function `docker_run`)
-  - I don't think `docker_run` is main focus for containers for this package, its more how it uses Docker to launch custom environments that can be templated (e.g. rstudio, shiny etc.) and for parrallisation e.g. `library(future)`  using GCE VMs as endpoints. [@MarkEdmondson1234]
+- [`googleComputeEngineR`](https://cloudyr.github.io/googleComputeEngineR/) (function `gce_vm_template()`)
+  - To enable quick deployments of key R services such as RStudio and Shiny onto cloud virtual machines (VMs), this package utilises Dockerfiles to move the labour of setting up those services from the user to a premade Docker image.  For example, by specifying the template `template="rstudio"` in `gce_vm_template()/gce_vm()` an up to date RStudio Service image is launched.  Specifying `template="rstudio-gpu"` will launch an RStudio Server image with a GPU attached, etc.
```
Comment (MarkEdmondson1234) [inline manuscript.Rmd]: How much detail would you like here?

File: manuscript.Rmd [hunk 2]
```diff
@@ -320,9 +320,9 @@ That way, package builders can use this image as a base image for there testing
 
 - [dynwrap](https://github.com/dynverse/dynwrap_containers/blob/master/.travis.yml) [@rcannood]
   - For this project, we use travis-ci to build rocker-derived containers, test them, and only push them to docker hub (from travis-ci.org) if the integration tests succeed.
-- Google Cloud Build [@MarkEdmondson1234]
+- Google Cloud Build
   - https://cloud.google.com/cloud-build/
-  - For me this is what makes Docker containers viable, as it builds the Dockerfiles on each GitHub commit.  It couples with Google Container Registry to build private Docker images, for downstream applications.
+  - Cloud Build is another continuous intergation service that helps move the workload of building the Docker images to an online service.  Cloud Build can be set up to build the Dockerfiles on each GitHub commit or release.  This means you do not need to build the Docker images locally, which can tie up resources since Docker images can be several GBs and take a long time to compile. Google Cloud Build works alongside Google Container Registry to allow you to build private and public Docker images, which allows you to build up your own dependency graphs for downstream applications.
```
Comment (MarkEdmondson1234) [inline manuscript.Rmd]: Re #37

File: manuscript.Rmd (hunk 1 above)
Comment (nuest) [reply manuscript.Rmd]: This is sufficient.

**Comment by nuest** [comment]
Thanks @MarkEdmondson1234 !

I'll ping you (and everyone else) when I've added the missing contents for a final review.

**Pull request merged**